### ISSUE TITLE
Tomb Room

### ISSUE THREAD

**Issue by iarna** [iarna/javatar-maps#6]
Per Will Henstrom:
Tomb Room and its corresponding pull location (There are some lvls that have tomb rooms that require someone else to come and pull you out, but depending on the room you are in, it might have a different pull location —> if there is a anti-mag room with no doors (a tomb room), you could have a small # in the room.  Then in the pull room, you could have the same number, but with something different to note that it is the pull location) - I know this one is a tough one.  Just curious what can be done. (I included an example of a map someone did for our lvl15.  You can see the tombstones on rooms that are tomb rooms, and then broken tombstones where they get pulled from.  Each has a number on them to show where to pull from)

<img width="512" alt="Lvl15full" src="https://github.com/user-attachments/assets/66147f99-db3f-4197-996b-76ac4dd6da18">

**Comment by iarna** [comment]
Given that the map shows fewer pull rooms than tomb rooms, I take it that multiple tomb rooms can be linked to single pull room?

**Comment by jektorrent** [comment]
correct.  the broken tombstone 1 (pull room 1) is the same encounter for all rooms marked tomb 1.

**Comment by iarna** [comment]
@jektorrent What are the tombstones without a number on them? Inescapable tombs?

**Comment by jektorrent** [comment]
I believe they are just unknown by the mapmaker.  

When I find myself in one of those, it's an exhaustive search of all the pull rooms to discover the right one.
